src/main/java/reactor/ipc/netty/channel/NettyContextSupport.java

```diff
@@ -223,29 +188,6 @@ public static final void addEncoderAfterReactorCodecs(Channel channel, String na
 		ChannelHandler extractor = addExtractor.apply(handler);
 		boolean shouldAddExtractor = extractor != null;
 
-		if (exists) {
```

Comment:
same here, skipIfExist, extractorExists and this block are all part of the fix for #22